crates/telio-utils/src/local_interfaces.rs

```diff
@@ -0,0 +1,72 @@
+#[cfg(feature = "test-util")]
+use if_addrs::{IfAddr, Ifv4Addr, Interface};
+
+use ipnet::{Ipv4Net, PrefixLenError};
+use std::net::{IpAddr, Ipv4Addr};
+use thiserror::Error as TError;
+
+#[cfg_attr(any(test, feature = "mockall"), mockall::automock)]
+/// Trait to get IF Address
```

Comment:
The code is in such a good readable state that several comments are trivial. Either modify them so that they explicitly add extra information or just remove them.

Reply:
What I mean by "explicitly add extra information" is:
- how to use the code on a higher level
- explain its behaviour if the functionality is complex or isn't that obvious
- justifying the changes (eg: "This was changed due to X, see Y"
- some other info that's not implicit when reading the code